mxcube3/ui/serverIO.js

```diff
@@ -32,6 +32,10 @@ export default class ServerIO {
       }
     });
 
+    this.hwrSocket.on('beam_value_changed', (msg) => {
```

Comment:
Is this a kind of conflict with PR #271 ? It seems to me the same lines are added twice